editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -1033,9 +1033,17 @@ void TileSetEditor::_on_workspace_process() {
 
 void TileSetEditor::_on_workspace_overlay_draw() {
 
-	if (!tileset.is_valid() || !get_current_texture().is_valid())
+	if (!tileset.is_valid())
 		return;
 
+	if (!get_current_texture().is_valid()) {
+		Ref<Font> font = get_font("font", "Label");
+		Color color = get_color("font_color", "Editor");
+		Point2i offset(workspace_overlay->get_margin(Margin::MARGIN_LEFT), WORKSPACE_MARGIN.y);
+		workspace_overlay->draw_string(font, offset, String("Select texture to edit from the left"), color);
```

Comment:
Here as well.

Reply:
Same thing, you can use a string literal directly.